standalone/Makefile

```diff
@@ -1,2 +1,43 @@
-init :
+hosts = icmr/playground
```

Comment:
Oops, just one thought. Should the host be accepted as an argument? I'm hesitant to couple our implementation too tightly to the `icmr/playground`.

Reply:
We will want to run these tasks against different envs (staging, qa et al.) and need to supply the env name somehow.  Maybe we could call the `hosts` argument `env`? That might be an unnecessary abstraction though, given that it's just the hosts filename that the makefile needs.

Reply:
Yup, I think we're both on the same page regarding supporting multiple envs. My question is should we modify this script right now to accept `icmr/playground` as an argument, instead of hardcoding it here?

Reply:
Oh, sorry I misunderstood. This line adds `icmr/playground` as a default value. The expected call is:
```
make restart-sidekiq hosts=<hosts>
```

Reply:
Oh, amazing! In that case, carry on 👌